includes/class-progress.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Progress Loader for Press Sync.
+ *
+ * @since 0.1.2
+ */
+class Press_Sync_Progress {
+
+	/**
+	 * Parent plugin class.
+	 *
+	 * @var   Press_Sync
+	 * @since 0.1.0
+	 */
+	protected $plugin = null;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 0.1.0
+	 *
+	 * @param Press_Sync $plugin Main plugin object.
+	 */
+	public function __construct( $plugin ) {
+		$this->plugin      = $plugin;
+		$this->cli_enabled = class_exists( 'WP_CLI' ) ? true : false;
```

Comment:
`class_exists` already returns boolean so the ternary is superfluous here.

Reply:
Awesome observation. Will get that changed.